src/time/Waveform.hpp

```diff
@@ -114,6 +103,21 @@ class Waveform {
    * @return Order that may be used.
    */
   int computeUsedOrder(int requestedOrder, int numberOfAvailableSamples);
+
+  /**
+   * @brief Returns point the closest time stored in _timeStepsStorage that is after normalizedDt
+   *
+   * @param normalizedDt point in time
+   * @return double point in time after normalizedDt in _timeStepsStorage
+   */
+  double findTimeAfter(double normalizedDt);
+
+  /**
+   * @brief Get keys of _timeStepsStorage in ascending order. Starting from low to high.
```

Comment:
normalized?

Reply:
I added a bit more of documentation here: c818df6fbebebc1daab52707e85a89abe74dbda6